awx/main/management/commands/create_preload_data.py

```diff
@@ -47,7 +47,7 @@ def handle(self, *args, **kwargs):
                                              created_by=superuser)
                 Host.objects.create(name='localhost',
                                     inventory=i,
-                                    variables="ansible_connection: local",
+                                    variables="ansible_connection: local\nansible_python_interpreter: '{{ ansible_playbook_python }}'",
```

Comment:
What this means in practice is that users that have python3 as their system Python *and* a py2 ansible venv will have to explicitly specify `ansible_python_interpreter: '{{ ansible_playbook_python }}` as a hostvar if they want to use `ansible_connection: local` (or they can alternatively use `delegate_to:`)